src/video_core/rasterizer_accelerated.cpp

```diff
@@ -22,6 +24,8 @@ static Common::Vec3f LightColor(const Pica::LightingRegs::LightColor& color) {
     return Common::Vec3u{color.r, color.g, color.b} / 255.0f;
 }
 
+constexpr float VR_IPD  = 0.065f;
```

Comment:
You could get realtime IPD by calculating the distance between left and right eye positions.